api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -2874,6 +2881,13 @@ message AppSessionRequest {
   ];
 }
 
+// AWSSessionMetadata contains extra AWS metadata of an AWS session.
+message AWSSessionMetadata {
+  // AWSRoleSessionName is used to specify the username when assuming the AWS
+  // IAM role.
+  string AWSRoleSessionName = 1 [(gogoproto.jsontag) = "aws_role_session_name,omitempty"];
```

Comment:
Could this be just role_session_name?

Also, could we use "proper" proto style for new messages?

- https://protobuf.dev/programming-guides/style/#message-field-names